source/one-or-more.d.ts

```diff
@@ -0,0 +1,22 @@
+/**
+Either matches an array containing only `Value`, or `Value` itself, not in an array.
+
+This can prove to be useful when you're not sure if you're going to get a type as an array, or on it's own.
```

Comment:
This is pretty much just repeating the above. In **what exact situations** can it be useful. Some examples would be nice.

Reply:
Dude there's an example there